src/Components/ServiceScene/Breakdowns/LabelValuesBreakdownScene.tsx

```diff
@@ -101,6 +120,51 @@ export class LabelValuesBreakdownScene extends SceneObjectBase<LabelValueBreakdo
         }
       }
     }
+
+    // If we're in an error state
+    if (newState.data?.state === LoadingState.Error && this.activeLayoutContainsNoPanels()) {
+      const activeLayout = this.getActiveLayout();
+      // And the active layout is not showing panels
+      if (activeLayout instanceof ByFrameRepeater) {
+        const errorState = this.getErrorStateAlert(newState.data.errors);
+        // Replace the loading or error state with new error
+        activeLayout.state.body.setState({
+          children: [errorState],
+        });
+      }
+    }
+  }
+
+  private getActiveLayout(): ByFrameRepeater | SceneFlexLayout | undefined {
+    const layoutSwitcher = this.state.body;
+    const activeLayout = layoutSwitcher?.state.layouts.find((layout) => layout.isActive);
+    if (activeLayout instanceof ByFrameRepeater || activeLayout instanceof SceneFlexLayout) {
+      return activeLayout;
+    }
+    return undefined;
+  }
+
+  private activeLayoutContainsNoPanels(): boolean {
+    const activeLayout = this.getActiveLayout();
+
+    if (activeLayout instanceof ByFrameRepeater) {
```

Comment:
No SceneFlexLayout? See line 141.

Reply:
Yeah the single view doesn't count as panels IMO as it will always render something even without dataframe

Reply:
but this should be documented at the very least, updating

Reply:
to clarify, since the single view panel is always rendered, even if no data is received but an error is, the error shows up in the UI and persists until the a request without an error is received. The row/grid views can have an error without any panels, so previously there would be no indication of anything being wrong.

Although now that I think about this while the single view should be fine as it will show a loading state when new queries are fired, we might want to re-add a loading state to the grid/rows when only the error message is visible.

Reply:
A bit of an edge case, created an issue for it tho: https://github.com/grafana/explore-logs/issues/785